graylog2-web-interface/src/views/components/searchbar/TimeRangeDropdownButton.test.tsx

```diff
@@ -36,9 +38,19 @@ jest.mock('views/stores/SearchConfigStore', () => ({
 }));
 
 describe('TimeRangeDropdownButton', () => {
+  type SUTProps = Partial<TimeRangeDropdownButtonProps> & {
```

Comment:
You could also use something like this:
```suggestion
  type SUTProps = Partial<React.ComponentProps<typeof TimeRangeDropdownButton>> & {
```

This makes it unnecessary to export/import the props type from the component.